docs/faq.adoc

```diff
@@ -15,14 +15,29 @@ Tips for fixing installation, syntax, processing, and rendering problems when us
 // :imagesdir: ../images
 :includedir: _includes
 
-== Frequently Asked Questions
+
+// == Frequently Asked Questions
+
+:q1: Does Asciidoctor only encode in US-ASCII?
+:q1a: How do I enable syntax highlighting in the deck.js backend?
+:q2: How do I make relative links to other AsciiDoc files to work correctly on GitHub?
+:q3: What is the internet mime type (aka MIME type) for AsciiDoc?
+:q4: Part way through the document, the blocks stop rendering correctly. What went wrong?
+:q5: Why don't URLs containing underscores (`_`) or carets (`{caret}`) work after they're rendered?
+
+. xref:faq1[_{q1}_]
+. xref:faq1a[_{q1a}_]
+. xref:faq2[_{q2}_]
+. xref:faq3[_{q3}_]
+. xref:faq_rendering[_{q4}_]
+. xref:faq_urls[_{q5}_]
```

Comment:
I like this. Perhaps we should add a horizontal rule between the question list and the body. (Three hyphens).